docs/controller.md

```diff
@@ -108,5 +108,17 @@
 - **bridge-stop**
   - **action**: stop the bridge
 
+- **regtest-mine-blocks**
+  - **action**: mine specified amount of blocks, optionally by a specified address
```

Comment:
What do you mean "by"? You mean where the coinbase reward goes to?

Reply:
Yes, that this address will receive the newly mined BTC